src/Test/Test.csproj

```diff
@@ -15,7 +15,7 @@
     </ItemGroup>
 
     <ItemGroup>
-        <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.0.0" />
+        <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.2.0" />
```

Comment:
is this a required change?

Reply:
Yeah, otherwise it doesn't have the appropriate win-arm files to load. My understanding is this isn't a breaking change.

If you revert the change and run test on a win-arm machine you get

```  
Starting test execution, please wait...
  A total of 1 test files matched the specified pattern.
EXEC : Testhost process exited with error : Unhandled exception. System.BadImageFormatException: Could not load file or
 assembly 'C:\runner\src\Test\bin\Release\net6.0\testhost.dll'. An attempt was made to load a prog
ram with an incorrect format. 
```